include/map.h

```diff
@@ -135,8 +148,8 @@ typedef struct StaticNPC {
     /* 0x028 */ s8 dropFlags;
     /* 0x029 */ s8 itemDropChance; // %
     /* 0x02A */ ItemDrop itemDrops[8];
-    /* 0x05A */ StatDrop heartDrops[8];
-    /* 0x09A */ StatDrop flowerDrops[8];
+    /* 0x05A */ StatDrop[5] heartDrops;
```

Comment:
We've been putting variable sizes after the name

Reply:
oh, whoops, typo

Reply:
I was going to change `typedef StatDrop -> StatDrops[5]` but decided against it